src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService x = new UserService();
+        User y = x.findByEmail(email);
+        if (y != null) {
+            if (y.getPassword().equals(password)) {
+                return true;
+            } else {
+                return false;
+            }
+        }
         return false;
```

Comment:
Let's try shorting this code. You can do the same without any if blocks. Try to use a ternary operator where needed